wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -720,6 +720,10 @@ struct ConversationEventPayloadProcessor {
         {
             conversation.mlsGroupID = mlsGroupID
         }
+
+        if let ciphersuite = payload.cipherSuite {
+            conversation.ciphersuite = MLSCipherSuite(rawValue: Int(ciphersuite))
+        }
```

Comment:
question: why do we store the ciphersuite in the conversation?

Reply:
It will be important later when conversations can have different cipersuites. In theory we could also get this information from CC but they don't expose it atm.